scripts/justmat/foulplay.lua

```diff
@@ -86,8 +80,10 @@ local BeatClock = require 'beatclock'
 
 local clk = BeatClock.new()
 
+local ALT = 0
```

Comment:
no biggie but curious why the screaming caps?

Reply:
Haha, no reason at all. I will turn down the volume now :)